### ISSUE TITLE
Trigger a release

### ISSUE THREAD

**Issue by danimo** [ngoduykhanh/PowerDNS-Admin#990]
The changelog between the last stable release and master is impressive, and esp. in terms of security (privilege fixes), IDN and API.

Security fixes should always trigger a release, but this seems release-worthy in general...

**Comment by jbe-dw** [comment]
@ngoduykhanh I've asked many times. I have written an ansible collection 6 month ago, and it was broken because we made changes on the API and there were no release since (my installation follows master and I have surprises at deployment).
Please take a break to fix a new release.

**Comment by danimo** [comment]
Also if you feel that others can help to lessen your burden, please let us know.

**Comment by jbe-dw** [comment]
I've contributed to this project, and it's not actively updated by the owner, it's mainly updated with proposed PRs. He's busy and sometime takes time to review and merge, but the frequency is pretty random.

It may be a good idea to open the project to a limited board. My feeling is that this software has the power to be a very good frontend to powerdns (I chose it over others mainly because PDNS lacks supporting RBAC rules for the API. The alternatives either lacks of an API to manage the keys or gravity.io for example would need a bit of configuration to do so).
